serverless/src/getAccessMethods/__tests__/handler.test.js

```diff
@@ -1,16 +1,66 @@
+import knex from 'knex'
+import mockKnex from 'mock-knex'
+
 import getAccessMethods from '../handler'
 import * as getJwtToken from '../../util'
 import * as getOptionDefinitions from '../getOptionDefinitions'
 import * as getServiceOptionDefinitions from '../getServiceOptionDefinitions'
+import * as getVerifiedJwtToken from '../../util/getVerifiedJwtToken'
+import * as getUsernameFromToken from '../../util/getUsernameFromToken'
+import * as getDbConnection from '../../util/database/getDbConnection'
+
+let dbConnectionToMock
+let dbTracker
 
+const OLD_ENV = process.env
```

Comment:
I don't see any overwriting of the `process.env` values in this test, you can remove all the references to this process in this test file.